# task_test_error_during

This is a test task, meant to be run before `task_test_after`.

Any explanation of the R logic, library dependencies, databases, or other
complexities should all be documented here.

The main purpose of this task is to test what happens when the process
throws the kind of errors we'd expect to see.